includes/helper/class-wp-job-manager-helper.php

```diff
@@ -85,11 +93,64 @@ public function init() {
 	 * Initializes admin-only actions.
 	 */
 	public function admin_init() {
+		$this->load_language_pack_helper();
 		add_action( 'plugin_action_links', [ $this, 'plugin_links' ], 10, 2 );
 		add_action( 'admin_notices', [ $this, 'licence_error_notices' ] );
 		$this->handle_admin_request();
 	}
 
+	/**
+	 * Load the language pack helper.
+	 */
+	private function load_language_pack_helper() {
+		if ( $this->language_pack_helper ) {
+			return;
+		}
+
+		$this->language_pack_helper = new WP_Job_Manager_Helper_Language_Packs( $this->get_plugin_versions(), $this->get_site_locales() );
+	}
+
+	/**
+	 * Get the versions for the installed managed plugins, keyed with the plugin slug.
+	 *
+	 * @return string[]
+	 */
+	public function get_plugin_versions() {
```

Comment:
I think this method could be `private` for now, right?

Reply:
Changed in cce6c6a24f1d7302c926e668155573af70c70260